AdminUi/packages/admin_api_types/lib/src/identity_deletion_process_auditlog_entries/identity_deletion_process_auditlog_entry.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'identity_deletion_process_auditlog_entry.g.dart';
+
+@JsonSerializable()
+class IdentityDeletionProcessAuditLogEntry {
+  final String id;
+  final DateTime createdAt;
+  final String messageKey;
+  final String? oldStatus;
+  final String newStatus;
+  final Map<String, String> additionalData;
+
+  IdentityDeletionProcessAuditLogEntry({
+    required this.id,
+    required this.createdAt,
+    required this.messageKey,
+    required this.newStatus,
+    required this.additionalData,
+    this.oldStatus,
+  });
+
+  factory IdentityDeletionProcessAuditLogEntry.fromJson(dynamic json) => _$IdentityDeletionProcessAuditLogEntryFromJson(json as Map<String, dynamic>);
+  Map<String, dynamic> toJson() => _$IdentityDeletionProcessAuditLogEntryToJson(this);
+}
+
+@JsonSerializable()
+class AdditionalData {
+  final String key;
+  final String value;
+
+  AdditionalData({
+    required this.key,
+    required this.value,
+  });
+
+  factory AdditionalData.fromJson(dynamic json) => _$AdditionalDataFromJson(json as Map<String, dynamic>);
+  Map<String, dynamic> toJson() => _$AdditionalDataToJson(this);
+}
```

Comment:
I don't see this is used anywhere?

Reply:
![image](https://github.com/user-attachments/assets/29bfb062-4a75-4204-a7f8-ff89e6f67aab)

It is used in here 
![image](https://github.com/user-attachments/assets/568883e6-6d60-44a2-be19-9acb37030441)
it will be displayed when identity is actually deleted.

Reply:
![image](https://github.com/user-attachments/assets/3c4a5484-21b7-46db-a2de-88ac202472c4)

Reply:
I cant see the class used anywhere.